RUN_ANALYSIS.py

```
@@ -0,0 +1,95 @@
+    # Создаём текстовый отчёт
+    print("\n[3/3] Creating text report...")
+    subprocess.run([
+        sys.executable, "save_text_report.py",
+        str(video_folder)
+    ])
+    
+    if video_file.exists():
+        print("\n" + "=" * 80)
+        print("COMPLETE!")
+        print("=" * 80)
+        print(f"\nCreated:")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)